Parser/convertSBEcnvVar.m

```diff
@@ -121,13 +121,16 @@
     % fluorescence (ug/l)
     case 'flC'
       name = 'CPHL';
-      comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements. Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
+      comment = getCPHLcomment('unknown','430nm','685nm');
 
     %fluorescence (ug/l)
     %Aquatracka SBE19plus cdom sensor as cphl implemented on #550
     case 'flCUVA'
+
+      warning('Assuming `flCUVA` as from Aquatracka III instead of Aquatracka UV');
```

Comment:
I tend not to use warning() for this kind of message. Instead something like:
```Matlab
fprintf('%s\n', ['Warning : ' 'Assuming blablablah']);
```

Reply:
@gg, this makes no sense for me (and it's an anti-pattern). Any deeper reason? 

The warning message goes to the same location as the fprintf call (stdout) anyway. Is this just because the user can "warning('off')"?

Reply:
I cannot remember right now. Mostly is a comment for consistency with what's already in the code elsewhere.